CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## v0.5.0 [2022-01-27]
+
+_Enhancements_
+
+- Added limit handling to `okta_application` table to reduce page size if `limit` is specified in the query while listing applications ([#61](https://github.com/turbot/steampipe-plugin-okta/pull/61))
```

Comment:
@misraved Since we did this for multiple tables now, we could probably generalize this note?

Reply:
We could also include notes about updating default page sizes and context cancellation handling